std_msgs/Header header

# Anchor to Anchor distance in m.
# 0: Anchor 0 to 1
# 1: Anchor 0 to 2
# 2: Anchor 1 to 2
bool[3] valid
float32[3] distance